lib/greenhouse_io/api/client.rb

```diff
@@ -93,12 +93,12 @@ def sources(id = nil, options = {})
       get_from_harvest_api "/sources#{path_id(id)}", options
     end
 
+    def custom_field(id = nil, options = {})
+      get_from_harvest_api "/custom_field#{path_id(id)}", options
+    end
+
     def custom_fields(field_type = nil, options = {})
-      if field_type
-        get_from_harvest_api "/custom_field#{path_id(field_type)}", options
-      else
-        get_from_harvest_api "/custom_fields", options
-      end
+      get_from_harvest_api "/custom_fields#{path_id(field_type)}", options
```

Comment:
Almost feel like we should rename `path_id` to `path_param` or something?
```rb
def path_param(param = nil)
  param.present? "/#{param}" : ''
end
```

Reply:
Wait is that it?
Calling `client.custom_fields(nil, { page: 12 })` before should have worked then no?
That would have done:
`get_from_harvest_api "/custom_fields", { page: 12 }`
What's changed for the pagination here?

Reply:
yeah, so nothing actually changed with pagination. Their API kinda sucks and just always returns the first page if there is only 1 page.

Reply:
> Almost feel like we should rename path_id to path_param or something?

Also, as far as renaming, I'm 100% up for it. I am trying to keep things as close to the OG code as possible, but IDK if that is really needed

Reply:
Sounds good yeah keep it that way 👍

Reply:
> So... we're still screwed? 🤔
I don't think so, I think there is a way around this, but I gotta investigate a bit more.